test/test_wahba_valenti.py

```diff
@@ -19,7 +15,7 @@ def test_q_acc_points_down(self):
             a1 = q.rotate(d)
             np.testing.assert_almost_equal(a0, a1, decimal=13)
 
-            q  = pqv.dq_acc(a0)
+            q  = pqv.q_acc(a0)
```

Comment:
IIRC, these are different functions.

Reply:
where's dq_acc declared? I see valenti_q_acc imported through the base __init__ , which is laundered through wahabi.valenti to be q_acc. 

```
from pyquat import valenti_q_mag as q_mag
from pyquat import valenti_q_acc as q_acc
```
but I don't see dq used elsewhere in the code, is that renamed elsewhere in an import?

these are getting hard to follow.

Reply:
I think we settled this during our zoom meeting yesterday. Two additional import lines should be added:

```
from pyquat import valenti_dq_mag as dq_mag
from pyquat import valenti_dq_acc as dq_acc
```

and the above change should be reverted.